<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QComboBox" name="reportsButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>ОТЧЕТЫ</string>
        </property>
        <item>
         <property name="text">
          <string>ПО ТИПАМ ОПЛАТ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ПО ОБЪЕКТАМ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ПО РАСХОДАМ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ПО КЛИЕНТАМ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ПО КВАРТИРАМ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ДОЛГИ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ОБЩИЙ</string>
         </property>
        </item>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="EcoTaxi">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>ERA CONSTRUCTION</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="settingsButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="placeholderText">
         <string>НАСТРОЙКИ</string>
        </property>
        <item>
         <property name="text">
          <string>ОБЪЕКТЫ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>РАСХОДЫ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>КЛИЕНТЫ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ТИП ОПЛАТЫ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ПОЛЬЗОВАТЕЛИ</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="dateButton">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>00.00.0000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>1156</width>
          <height>520</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>80</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
             <item>
              <widget class="QLabel" name="clientLabel">
               <property name="text">
                <string>Клиент:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="clientBox">
               <property name="maximumSize">
                <size>
                 <width>319</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
             <item>
              <widget class="QLabel" name="objectLabel">
               <property name="text">
                <string>Объект:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="objectBox"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
             <item>
              <widget class="QLabel" name="apartmentLabel">
               <property name="text">
                <string>Квартира:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="apartmentBox"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,2">
             <property name="spacing">
              <number>7</number>
             </property>
             <item>
              <widget class="QLabel" name="costLabel">
               <property name="text">
                <string>Стоимость:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="costEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
             <item>
              <widget class="QLabel" name="discountLabel">
               <property name="text">
                <string>Скидка:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="discountEdit">
               <property name="placeholderText">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
             <item>
              <widget class="QLabel" name="totalLabel">
               <property name="text">
                <string>Итого:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="totalEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,2">
             <item>
              <widget class="QLabel" name="paymentTypeLabel">
               <property name="text">
                <string>Тип оплаты:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="paymentTypeBox"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,2">
             <item>
              <widget class="QLabel" name="currencyLabel">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Валюта:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="currencyEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="1,2">
             <item>
              <widget class="QLabel" name="installmentLabel">
               <property name="text">
                <string>Рассрочка:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="installmentCheckBox">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="2,3">
               <property name="spacing">
                <number>7</number>
               </property>
               <item>
                <widget class="QLabel" name="initialPaymentLabel">
                 <property name="text">
                  <string>Первоначальный взнос:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="initialPaymentEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="2,3">
               <property name="spacing">
                <number>7</number>
               </property>
               <item>
                <widget class="QLabel" name="monthQuantityLabel">
                 <property name="text">
                  <string>Сколько месяцев:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="monthQuantityEdit">
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                 <property name="placeholderText">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="2,3">
               <property name="spacing">
                <number>7</number>
               </property>
               <item>
                <widget class="QLabel" name="monthlyPaymentLabel">
                 <property name="text">
                  <string>Каждый месяц:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="monthlyPaymentEdit">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1,0">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item alignment="Qt::AlignBottom">
                <widget class="QLabel" name="descriptionLabel">
                 <property name="text">
                  <string>Описание:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextEdit" name="descriptionEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>100</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="addEventButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>52</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Добавить</string>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="version">
               <property name="text">
                <string>0.0.8</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="spacing">
       <number>40</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="paymentsButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="text">
         <string>ОПЛАТА</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="expensesButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="text">
         <string>РАСХОДЫ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="salesButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="text">
         <string>ПРОДАЖИ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
